src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinemaHalls")
```

Comment:
better table and column naming in SQL using '_' -> cinema_halls

Reply:
done